extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AmazonLambdaRecorder.java

```diff
@@ -49,7 +50,8 @@ public void setHandlerClass(Class<? extends RequestHandler<?, ?>> handler, BeanC
         beanContainer = container;
         AmazonLambdaRecorder.objectMapper = getObjectMapper()
                 .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-                .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true);
+                .configure(MapperFeature.ACCEPT_CASE_INSENSITIVE_PROPERTIES, true)
+                .registerModule(new JodaModule());
```

Comment:
I let's go with this for now, but I think we'll want to introduce an `ObjectMapperCustomizer` to the module that takes care of all this